src/test/sheet-utils.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const sheetUtils = require('../main/lib/sheet-utils');
+
+describe('sheet-utils', () => {
+  describe('groupToSheets', () => {
+    it('should group the paths by top level directory', function () {
+      const paths = [
+        'first/file.json',
+        'first/another.json',
+        'second/file.json'
+      ];
+
+      const grouped = sheetUtils.groupToSheets(paths);
+      assert.equal(Object.keys(grouped).length, 2);
+      assert.ok(hasProperty(grouped, 'first'));
```

Comment:
For me `hasProperty` is bit vague. I'd like to know exactly that there is such property and that it has no fragments.

Reply:
Instead off this you could create another assertion function e.g.

`assertGroupExists('first', ['file.json', 'another.json'])`
`assertGroupExists('second', ['file.json'])`

and finally in there test:

`assertGroupExists('another.json', [])`

Reply:
This way you hide complexity and repetition of asserting right structure.